tests/chart_tests/test_authsidecar.py

```diff
@@ -271,3 +396,65 @@ def test_authSidecar_houston_configmap_with_securityContext(self, kube_version):
+        """Test All Services with authSidecar and set no values in ingressAllowedNamespaces.
+        Only include networkpolicies that have the network.openshift.io/policy-group: ingress label."""
+        docs = render_chart(
+            kube_version=kube_version,
+            values={"global": {"authSidecar": {"enabled": True, "ingressAllowedNamespaces": []}}},
+            show_only=[
+                "charts/alertmanager/templates/alertmanager-networkpolicy.yaml",
+                "charts/astronomer/templates/astro-ui/astro-ui-networkpolicy.yaml",
+                "charts/astronomer/templates/houston/api/houston-networkpolicy.yaml",
+                "charts/astronomer/templates/registry/registry-networkpolicy.yaml",
+                "charts/grafana/templates/grafana-networkpolicy.yaml",
+                "charts/kibana/templates/kibana-networkpolicy.yaml",
+                "charts/prometheus/templates/prometheus-networkpolicy.yaml",
+            ],
+        )
+
+        assert len(docs) == 7
+
+        for doc in docs:
+            assert "NetworkPolicy" == doc["kind"]
+            namespaceSelectors = doc["spec"]["ingress"][0]["from"]
+            assert {"namespaceSelector": {"matchLabels": {"network.openshift.io/policy-group": "ingress"}}} in namespaceSelectors
+            assert {
+                "namespaceSelector": {"matchLabels": {"app.astronomer.io/allow-authsidecar-traffic": "true"}}
+            } in namespaceSelectors
+
+    def test_authSidecar_all_services_with_ingress_allowed_namespaces(self, kube_version):
+        """Test All Services with authSidecar and allow some traffic namespaces.
+        Only include networkpolicies that have the network.openshift.io/policy-group: ingress label"""
+
+        docs = render_chart(
+            kube_version=kube_version,
+            values={"global": {"authSidecar": {"enabled": True, "ingressAllowedNamespaces": ["astro", "ingress-namespace"]}}},
+            show_only=[
+                "charts/alertmanager/templates/alertmanager-networkpolicy.yaml",
+                "charts/astronomer/templates/astro-ui/astro-ui-networkpolicy.yaml",
+                "charts/astronomer/templates/houston/api/houston-networkpolicy.yaml",
+                "charts/astronomer/templates/registry/registry-networkpolicy.yaml",
+                "charts/grafana/templates/grafana-networkpolicy.yaml",
+                "charts/kibana/templates/kibana-networkpolicy.yaml",
+                "charts/prometheus/templates/prometheus-networkpolicy.yaml",
+            ],
+        )
+
+        assert len(docs) == 7
+
+        for doc in docs:
+            assert "NetworkPolicy" == doc["kind"]
+            namespaceSelectors = doc["spec"]["ingress"][0]["from"]
+            assert {"namespaceSelector": {"matchLabels": {"network.openshift.io/policy-group": "ingress"}}} in namespaceSelectors
+            assert {
+                "namespaceSelector": {"matchLabels": {"app.astronomer.io/allow-authsidecar-traffic": "true"}}
```

Comment:
We should remove this `app.astronomer.io/allow-authsidecar-traffic` label here since we decided to go with `ingressAllowedNamespaces`. Sometimes people use tests as indicators of features that can be enabled, and it would be unfortunate if somebody saw this and went looking for this feature.